### DISCUSSION TITLE
Remove Slow Loading (Optimize in websites)

### DISCUSSION THREAD

**Discussion by MAAZQAZI** [splinetool/react-spline#74] [Ideas]
It causes the website to slow like turtle. you should optimize it.  I  ❤  spline.

**Comment by marcofugaro** [comment]
You can optimize your scene using the `3D Vitals` diagnostics.

<img width="283" alt="image" src="https://user-images.githubusercontent.com/7217420/186629430-7d3d0dfd-22f0-4d71-9402-ad599ecece9b.png">

**Reply by ningshunqiang** [reply]
3D Vitals what is it, can you post a link?

**Reply by marcofugaro** [reply]
Here https://docs.spline.design/f6351697797e4e41bbf57d62ab905a06
